Modules/quote/Model/Quote.php

```diff
@@ -51,10 +51,11 @@ public function getAllInfo($id_space, $id) {
 
             $data["recipient"] = $modelUser->getUserFUllName($data["id_user"]);
             $resps = $modelUserClient->getUserClientAccounts($data["id_user"], $id_space);
-            if (count($resps) > 0) {
+            if ($resps && !empty($resps)) {
                 $data["address"] = $modelClient->getAddressInvoice($id_space, $resps[0]["id"]);
```

Comment:
On ne l a pas ds client?

Reply:
No, the sql request is a join with cl_addresses table